types/fsp_test.go

```diff
@@ -16,119 +16,119 @@ package types
 
 import (
 	"strconv"
+	"testing"
 
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = Suite(&FspTest{})
-
-type FspTest struct{}
-
-func (s *FspTest) TestCheckFsp(c *C) {
-	c.Parallel()
+func TestCheckFsp(t *testing.T) {
+	t.Parallel()
 	obtained, err := CheckFsp(int(UnspecifiedFsp))
-	c.Assert(obtained, Equals, DefaultFsp)
-	c.Assert(err, IsNil)
+	require.Equal(t, obtained, DefaultFsp)
```

Comment:
Expected is the second argument. Please fix it.

Reply:
https://github.com/pingcap/tidb/pull/28044#discussion_r708296079
Done.

Reply:
> Expected is the second argument. Please fix it.

Done.